packages/playground-examples/copy/ja/4-0/New JS Features/Logical Operators and Assignment.ts

```diff
@@ -0,0 +1,41 @@
+// 論理代入演算子は、
```

Comment:
[MDN](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment)が 論理和代入なので、それに沿っておきますか
```suggestion
// 論理和代入演算子は、
```

Reply:
論理和代入は`||=`を指しますが、ここは複数形なので、
`||=`/`&&=`/`??=`のグループのことだと思います。

Reply:
たしかに！